patches/api/0327-Expose-tracked-entities.patch

```diff
@@ -0,0 +1,27 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: SuperKooks <superkooks@superkooks.com>
+Date: Wed, 4 Aug 2021 18:18:52 +1000
+Subject: [PATCH] Expose tracked entities
+
+
+diff --git a/src/main/java/org/bukkit/entity/Player.java b/src/main/java/org/bukkit/entity/Player.java
+index 5e8814cc317a705eaf8bdd9f3876a5366c0a0226..22749f6c83f60c2038b5f7ba14d0e6c16215d77c 100644
+--- a/src/main/java/org/bukkit/entity/Player.java
++++ b/src/main/java/org/bukkit/entity/Player.java
+@@ -2050,6 +2050,16 @@ public interface Player extends HumanEntity, Conversable, OfflinePlayer, PluginM
+     Set<Player> getTrackedPlayers();
+     // Paper end
+ 
++    // Paper start
++    /**
++     * Gets entities within this player's tracking range (that the player's client can "see")
++     * @param entityClass The class of entities to filter for
++     * @return Returns the filtered set of entities
++     */
++    @NotNull
++    <T extends org.bukkit.entity.Entity> Set<T> getTrackedEntities(@Nullable Class<T> entityClass);
```

Comment:
Technically this would be more correct:
```suggestion
+    <T extends org.bukkit.entity.Entity> Set<? extends T> getTrackedEntities(@Nullable Class<? extends T> entityClass);
```